api/orchestration_movers.go

```diff
@@ -528,3 +528,44 @@ func (o *datasyncOrchestrator) datamoverRunDescribe(ctx context.Context, group,
 		Result:                   exec.Result,
 	}, nil
 }
+
+// startTaskRun starts the execution for a given task
+func (o *datasyncOrchestrator) startTaskRun(ctx context.Context, group, name string) (*datasync.StartTaskExecutionOutput, error) {
+
+	task, _, err := o.taskDetailsFromName(ctx, group, name)
+	if err != nil {
+		return nil, err
+	}
+	if *task.Status == "AVAILABLE" {
+		out, err := o.datasyncClient.StartTaskExecution(ctx, aws.StringValue(task.TaskArn))
+		if err != nil {
+			return nil, err
+		}
+
+		arn := out.TaskExecutionArn
+		parts := strings.Split(*arn, "/")
+		arn = &parts[len(parts)-1]
+		out.TaskExecutionArn = arn
+
+		return out, nil
+	}
+	return nil, apierror.New(apierror.ErrConflict, "another datasync mover task is already running", nil)
+}
+
+// stopTaskRun starts the execution for a given task
+func (o *datasyncOrchestrator) stopTaskRun(ctx context.Context, group, name string) (*datasync.CancelTaskExecutionOutput, error) {
```

Comment:
I don't think we care about `CancelTaskExecutionOutput` so this func can just return `error`

Reply:
Fixed.